tour/authorizations-code.md

```diff
@@ -6,52 +6,63 @@ Below is a solution for the exercise.
+  mutation1.put("hero", "wearsCape?", "true");
+  
+  Mutation mutation2 = new Mutation("id0002");
+  mutation2.put("hero", "alias", "Robin");
+  mutation2.put("hero", "name", colVis, "Dick Grayson");
+  mutation2.put("hero", "wearsCape?", "true");
+  
+  Mutation mutation3 = new Mutation("id0003");
+  mutation3.put("villain", "alias", "Joker");
+  mutation3.put("villain", "name", "Unknown");
+  mutation3.put("villain", "wearsCape?", "false");
 
-    // Create a BatchWriter to the GothamPD table and add your mutations to it.
-    // Once the BatchWriter is closed by the try w/ resources, data will be available to scans.
-    try (BatchWriter writer = client.createBatchWriter("GothamPD")) {
-        writer.addMutation(mutation1);
-        writer.addMutation(mutation2);
-        writer.addMutation(mutation3);
-    }
+  // Create a BatchWriter to the GothamPD table and add your mutations to it.
+  // Once the BatchWriter is closed by the try w/ resources, data will be available to scans.
+  try (BatchWriter writer = client.createBatchWriter("GothamPD")) {
+    writer.addMutation(mutation1);
+    writer.addMutation(mutation2);
+    writer.addMutation(mutation3);
+  }
 
-    // Create a second client for the commissioner. Then print all the rows visibile to the
-    // commissioner. Make sure to pass the proper authorizations to the Scanner
-    try (AccumuloClient commishClient = Accumulo.newClient().from(client.properties())
-            .as("commissioner", "gordonrocks").build();
-        Scanner scan = commishClient.createScanner("GothamPD", auths)) {
-        System.out.println("Gotham Police Department Persons of Interest:");
-        for (Map.Entry<Key, Value> entry : scan) {
-            System.out.printf("Key : %-60s  Value : %s\n", entry.getKey(), entry.getValue());
-        }
-    }
+  // Create a second client for the commissioner user and output all the rows visible to them.  
+  // Make sure to pass the proper authorizations.
+  try (AccumuloClient commishClient = Accumulo.newClient().from(client.properties()).as("commissioner", "gordonrocks").build()){
+    Scanner scan = commishClient.createScanner("GothamPD", auths);
+    System.out.println("Gotham Police Department Persons of Interest:");
+    for (Map.Entry<Key, Value> entry : scan) {
+	  System.out.printf("Key : %-50s  Value : %s\n", entry.getKey(), entry.getValue());
+	 }
+  }
 }
+    // Create a second client for the commissioner user and output all the rows visible to them.  
+	// Make sure to pass the proper authorizations.
+	    try (AccumuloClient commishClient = Accumulo.newClient().from(client.properties()).as("commissioner", "gordonrocks").build()){
+	    	Scanner scan = commishClient.createScanner("GothamPD", auths);
+	    	System.out.println("Gotham Police Department Persons of Interest:");
+	    	for (Map.Entry<Key, Value> entry : scan) {
+	            System.out.printf("Key : %-50s  Value : %s\n", entry.getKey(), entry.getValue());
```

Comment:
This code block could be formatted, and should not include tab characters.

Reply:
I believe what happened here is that I accidentally left out the this file when I was committing the second time. The version I have right now gets rid of the tab characters. Will send up a third commit soon.